docs/design/build-system.md

```diff
@@ -72,8 +104,9 @@ Stylesheet third party dependencies are treated mostly the same as sources.
 JavaScript third party dependencies suffer a more involved process.
 They are first resolved to a folder in `node_modules` via [Node Module Resolution](https://nodejs.org/api/modules.html#modules_modules).
 A given module might have several different entry points, for instance one for use in NodeJS and another one for using in the browser.
-Each entry point is listed in under a name pointing at a js file in that module's `package.json`.
-We use `es2015 > browser > module > main` a priority list, where the first key matched name determines which entry point to use.
+Although `package.json` only officially supports listing one entry point under the `main` key, it's common for npm packages to list [other entry points](https://2ality.com/2017/04/setting-up-multi-platform-packages.html).
```

Comment:
Addresses https://github.com/angular/angular-cli/pull/16400#discussion_r356715022, cc @dgp1130.

Reply:
Makes sense, thanks for the link.